tutorials/content/procedural_geometry.rst

```diff
@@ -0,0 +1,144 @@
+.. _doc_procedural_geometry:
```

Comment:
This was added by reduz in 131c90b6, so needs checking to ensure it works in 3.1.x too.

Reply:
Had a quick look, all the methods used here seem to be bound in `3.1-stable`.